spacy/lang/fr/lemmatizer/_adjectives_irreg.py

```diff
@@ -51186,5 +51184,5 @@
     "zygotique": ("zygotique",),
     "zymogène": ("zymogène",),
     "zymonucléique": ("zymonucléique",),
-    "zymotique": ("zymotique",)
+    "zymotique": ("zymotiqueg",)
```

Comment:
I guess there is a small mistake here with "zymotiqueg"

Reply:
yes thanks! :) Corrected.